package.json

```diff
@@ -54,59 +55,87 @@
     "@angular/platform-browser-dynamic": "14.0.2",
     "@angular/router": "14.0.2",
     "@nrwl/angular": "14.3.5",
+    "@sampullman/fetch-api": "^0.7.0",
+    "date-fns": "^2.28.0",
     "localforage": "^1.10.0",
     "rxjs": "7.4.0",
     "tslib": "^2.0.0",
+    "vue": "^3.2.37",
+    "vue-i18n": "^9.1.6",
+    "vue-router": "^4.0.16",
     "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "14.0.2",
+    "@angular-devkit/build-angular": "^14.1.0",
     "@angular-eslint/eslint-plugin": "13.2.1",
     "@angular-eslint/eslint-plugin-template": "13.2.1",
     "@angular-eslint/template-parser": "13.2.1",
     "@angular/cli": "~14.0.0",
     "@angular/compiler-cli": "14.0.2",
     "@angular/language-service": "14.0.2",
+    "@babel/core": "^7.18.5",
+    "@babel/plugin-transform-modules-commonjs": "^7.18.2",
+    "@babel/preset-env": "^7.18.2",
     "@commitlint/cli": "^12.1.1",
     "@commitlint/config-conventional": "^12.1.1",
+    "@cypress/code-coverage": "^3.10.0",
+    "@cypress/vite-dev-server": "^3.0.0",
```

Comment:
I think those deps are causing resolution conflicts with nx cypress version

Reply:
yes, they are. do you have any suggestions on how to resolve this